app/scripts/lib/buy-url.js

```diff
@@ -157,11 +161,11 @@ export default async function getBuyUrl({ chainId, address, service }) {
     case 'wyre':
       return await createWyrePurchaseUrl(address, chainId);
     case 'transak':
-      return createTransakUrl(address, chainId);
+      return createTransakUrl(address, chainId, symbol);
```

Comment:
I just feel the usage of `account` as parameter could be more readable and representative than using `address` at it could be confusing with token address.